90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -1600,7 +1600,11 @@ load_be_cmdline() {
   # If a user-entered cmdline is found, it is not modified
   if [ -r "${BASE}/cmdline" ]; then
     zdebug "using ${BASE}/cmdline as commandline for ${zfsbe_fs}"
-    head -1 "${BASE}/cmdline" | tr -d '\n'
+
+    # root= is ALWAYS controlled by ZFSBootMenu
+    zfsbe_args="$( suppress_kcl_arg root "$(head -1 "${BASE}/cmdline" | tr -d '\n')" )"
+
+    echo "${zfsbe_args}"
```

Comment:
Why save a var to echo instead of just doing
```sh
suppress_kcl_arg root "$(head -1 "${BASE}/cmdline" | tr -d '\n')"
```

Reply:
Good call.